Revit_oM/Enums/OutlineAlignment.cs

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2023, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.ComponentModel;
+
+namespace BH.oM.Revit.Enums
+{
+    /***************************************************/
+
+    [Description("Direction of the sheet viewport alignment.")]
+    public enum OutlineAlignment
+    {
```

Comment:
Do you think we will ever have other distribution layouts? To me an enum feels a bit too constrained, I would consider an interface instead, for example:
- `IViewDistribution` interface (no properties? or maybe `Gap`)
- `DistributionInRows` class implementing `IViewDistribution` with properties that would define left to right, top to down etc.
- `DistributionInColumns` class implementing `IViewDistribution` with properties that would define left to right, top to down etc.
- distribution method would take `IViewDistribution`

...this is just an idea off the top of my head, so not sure if valid. Thinking out loud.